src/ui/server/helpers/summary-lists/generate-field-group-item/index.ts

```diff
@@ -2,6 +2,7 @@ import getKeyText from '../get-key-text';
 import { DEFAULT } from '../../../content-strings';
 import { objectHasProperty } from '../../object';
 import { SummaryListItemData, SummaryListItemDataInput } from '../../../../types';
+import transformNumberToString from '../../display-number-field-value';
```

Comment:
Can we align the name of the function with the name of the file ` transform-number-to-string`